huntsman/camera/pyro.py

```diff
@@ -0,0 +1,502 @@
+import sys
+import os
+import glob
+import time
+from warnings import warn
+from threading import Event
+from threading import Timer
+from threading import Thread
+from threading import Lock
+import subprocess
+
+from astropy import units as u
+import Pyro4
+
+from pocs.base import PanBase
+from pocs.utils import current_time
+from pocs.utils.logger import get_root_logger
+from pocs.utils import load_module
+from pocs.camera import AbstractCamera
+
+from huntsman.utils import load_config
+
+# Enable local display of remote tracebacks
+sys.excepthook = Pyro4.util.excepthook
+
+
+class Camera(AbstractCamera):
+    """
+    Class representing the client side interface to a distributed camera
+    """
+    def __init__(self,
+                 uri,
+                 name='Pyro Camera',
+                 model='pyro',
+                 *args, **kwargs):
+        super().__init__(name=name, port=uri, model=model, *args, **kwargs)
+
+        # Connect to camera
+        self.connect(uri)
```

Comment:
Probably a good idea, if for no other reason than to order to allow us to later on to defer the `connect()` instead of calling it from within `__init__()`.   That's something we might want to do, because it's the `connect()` that triggers the remote object creation and, for real cameras, the lengthy hardware initialisation process.